README.md

```diff
@@ -38,6 +38,31 @@ cluster on AWS.
 
 ------
 
+## Compatibility with Cluster API and Kubernetes Versions
+
+This provider's versions are compatible with the following versions of Cluster API:
+
+||Cluster API 0.1|
```

Comment:
Considering previous tags for cluster-api are 0.0.0-alpha.x, it should be safe to call the v1alpha1 cut of Cluster API v0.1. That said, are we more concerned with binary compatibility of the controllers or the API versions here? If it's the API versions, then we should probably tag and vendor 'v1alpha1'.

I've now confused myself a bit and am wondering if we should break it down into both API and binary version compatibility here and list both?

Reply:
I think API version is first priority re compatibility, followed by semver (if we need to point out a minimum semver b/c of bug fixes or whatnot). Does that sound good?

Reply:
In that case, we should probably call out `v1alpha1`, `v0.1` here